### PR TITLE
Update app code to enable single appsettings.json file

### PR DESCRIPTION
**Pull request by jlhogan**
requires adding more values to the deployment pipelines!

### CODE REVIEW

File: src/WingedKeys/appsettings.json [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
   "AllowedHosts": "*",
-  "BaseUri": "https://localhost:80",
+  "BaseUri": "https://localhost:5050",
   "CertificateFileName": "",
   "CertificatePassword": "",
   "ClientUris": "https://localhost:5001,https://localhost:5000",
+  "IssuerUri": "https://winged-keys:5050",
   "ConnectionStrings": {
-    "WingedKeys": "*"
+    "WingedKeys": ""
```
Comment (cailyncodes) [inline src/WingedKeys/appsettings.json]: I think this needs to be set to the development value, no?

Comment (jlhogan) [reply src/WingedKeys/appsettings.json]: no in dev that comes from the docker-compose env

Comment (jlhogan) [reply src/WingedKeys/appsettings.json]: which does not need to be the way it is but is how it is in hedwig

Comment (jlhogan) [reply src/WingedKeys/appsettings.json]: changed my mind, adding it here. will do in hedwig as well one day.

**Pull request merged**